chromium_src/chrome/browser/ui/tabs/tab_model.cc

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/tabs/tab_model.h"
+
+#include "brave/components/containers/core/common/features.h"
```

Comment:
everything related to containers should be under `BUILDFLAG(ENABLE_CONTAINERS)`, but I don't think you should mention the containers feature here at all. Just keep `partitioned_tab_visual_data_` getters/setter and remove `CHECK` calls.